app/src/main/res/layout/fragment_details.xml

```diff
@@ -1,127 +1,141 @@
+        android:layout_height="match_parent">
 
-    <TextView
-        android:paddingTop="30dp"
-        android:id="@+id/tvDetailTitle"
-        android:layout_below="@+id/ivShowPoster"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:layout_alignParentStart="true"
-        android:layout_alignParentLeft="true"
-        android:layout_alignParentEnd="true"
-        android:layout_alignParentRight="true"
-        android:gravity="center"
-        android:text="Show Title"
-        android:textAppearance="@android:style/TextAppearance.Large"
-        android:textSize="30sp" />
-
-    <TextView
-        android:id="@+id/tvYearAndNetwork"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:layout_below="@+id/tvDetailTitle"
-        android:layout_alignParentStart="true"
-        android:layout_alignParentLeft="true"
-        android:layout_alignParentEnd="true"
-        android:layout_alignParentRight="true"
-        android:gravity="center"
-        android:padding="2dp"
-        android:text="Year Aired | Network"
-        android:textSize="12sp"/>
-
-    <TextView
-        android:id="@+id/tvDetailOverview"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:layout_below="@+id/tvYearAndNetwork"
-        android:layout_alignParentStart="true"
-        android:layout_alignParentLeft="true"
-        android:layout_alignParentEnd="true"
-        android:layout_alignParentRight="true"
-        android:gravity="center"
-        android:padding="10dp"
-        android:text="Overview"
-        android:textSize="16sp"/>
+        <ImageView
+            android:id="@+id/ivShowPoster"
+            android:layout_width="match_parent"
+            android:layout_height="150dp"
+            android:layout_alignParentTop="true"
+            android:layout_marginTop="20dp"
+            android:gravity="center"
+            tools:srcCompat="@tools:sample/avatars" />
 
-    <LinearLayout
-        android:id="@+id/lRatings"
-        android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:orientation="vertical"
-        android:layout_below="@+id/tvDetailOverview"
-        android:layout_marginTop="15dp">
+        <TextView
+            android:id="@+id/tvDetailTitle"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_below="@+id/ivShowPoster"
+            android:layout_alignParentStart="true"
+            android:layout_alignParentLeft="true"
+            android:layout_alignParentEnd="true"
+            android:layout_alignParentRight="true"
+            android:fontFamily="@font/roboto_bold_italic"
+            android:gravity="center"
+            android:paddingStart="5dp"
+            android:paddingEnd="5dp"
+            android:paddingTop="30dp"
```

Comment:
Dp values like this are things that you'll ultimately want to save in dimens.xml files.  This will allow you to also easily change the sizes based on different screen sizes as well